README.md

```diff
@@ -2,6 +2,18 @@
 
 Substra end-to-end (e2e) tests
 
+# A few words regarding tests design
+
+- Must be fast to run
+- Should be stable: no random failure
```

Comment:
What about using the word "deterministic" here, e.g. "Should have a deterministic behavior (not fail randomly)"?

Reply:
Sounds good.